Source/RestoredState.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  RestoredState.swift
+//  RxBluetoothKit
+//
+//  Created by Kacper Harasim on 21.05.2016.
+//  Copyright © 2016 Polidea. All rights reserved.
+//
+
+import Foundation
+import CoreBluetooth
+
+/**
+ Convenience class which helps reading state of restored BluetoothManager
+ */
+public struct RestoredState {
+
+    public let restoredStateData: [String:AnyObject]
+
+    let bluetoothManager: BluetoothManager
```

Comment:
I'm thinking that it might be useful for user to have access to raw bluetooth manager, however that wouldn't be very pure...